### ISSUE TITLE
CreateProject crash on Mac

### ISSUE THREAD

**Issue by xpqz** [aplteam/Cider#85]
```
Dyalog APL/S-64 Version 19.0.50015
Serial number: UNREGISTERED - not for commercial use
+-----------------------------------------------------------------+
| Dyalog is free for non-commercial use but is not free software. |
| A basic licence can be used for experiments and proof of        |
| concept until the point in time that it is of value.            |
| For further information visit                                   |
| https://www.dyalog.com/prices-and-licences.htm                  |
+-----------------------------------------------------------------+
Tue Nov 19 11:54:41 2024
      ]CIDER.version
0.42.2+671

      ]CIDER.CreateProject work/mytest

"/Users/stefan/work/mytest" does not exist yet - create? (Y/n) Y
* Command Execution Failed: LENGTH ERROR: Boolean singleton value required
```

The problem seems to be 

Cider_UC:

```
      :If ~noEditFlag
          :If ¯1=P.ProjectConfig filename     ⍝ CRASH HERE
              1 ⎕NDELETE filename
              r←'Cancelled by user' ⋄ →0
          :EndIf
          config←⊃⎕NGET filename 1
          :If success←0<≢(∊config)~' '
              config←⎕JSON⍠('Dialect' 'JSON5')⊣¯1↓∊config,¨⎕UCS 10
          :Else
              ⎕NDELETE filename
              :Return
          :EndIf
```         
          
ProjectConfig:

```
 :Else
     r←↑ns.cider_config  ⍝ CRASH HERE
 :EndIf
```

**Comment by aplteam** [comment]
Fixed in 0.44.2

**Comment by aplteam** [comment]
Typo: fixed in 0.44.0
